rio_tiler/io/base.py

```diff
@@ -199,9 +202,9 @@ def parse_expression(self, expression: str) -> Tuple:
         _re = re.compile(assets.replace("\\\\", "\\"))
         return tuple(set(re.findall(_re, expression)))
 
-    def info(
+    def info(  # type: ignore
```

Comment:
we need the `type: ignore` here because the output is `Dict[str, Info]` while in the baseclass it's `Info` only